compiler/circle-mpqsolver/src/core/DataProvider.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __MPQSOLVER_DATA_PROVIDER_H__
+#define __MPQSOLVER_DATA_PROVIDER_H__
+
+#include <dio_hdf5/HDF5Importer.h>
+
+#include <luci/IR/Module.h>
+
+#include <string>
+#include <vector>
+
+namespace mpqsolver
+{
+namespace core
+{
+
+struct InputData
+{
+  std::vector<char> _data;
+
+  InputData(size_t size) : _data(size) {}
+};
```

Comment:
`_` prefix is often used for a private member. Can you use just a vector (remove this struct) or make the member private?

Reply:
> `_` prefix is often used for a private member. Can you use just a vector (remove this struct) or make the member private?

Ok.

Reply:
done